Aptoide/aptoide-client-v8
# [APP-431] Promotional Cards

**What does this PR do?**

This PR aims at implementing the new promotional cards card, coming soon card.
In this PR I also fixed "home_interact" (impression and click) events for all promotional cards. 

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] Home.java

**How should this be manually tested?**

For the analytics part, check that they are being correctly sent (impression and click on the respective items).
Regarding the promotional card, ask me for the rewrites so that you can see the cards. Then check that the card is correctly mapped, press "notify me" and check that you receive the notification for the app that was requested.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-431](https://aptoide.atlassian.net/browse/APP-431)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
